src/scss/fluid-bootstrap/_alert.scss

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2011-2018, Hortonworks Inc. All rights reserved.
+ * Except as expressly permitted in a written agreement between you
+ * or your company and Hortonworks, Inc, any use, reproduction,
+ * modification, redistribution, sharing, lending or other exploitation
+ * of all or any part of the contents of this file is strictly prohibited.
+ */
+
+.alert {
+  box-shadow: 1px 1px 5px 0 $fluid-gray-2;
+  font-size: $font-size-base * .75; // 12px;
+  
+  div {
```

Comment:
I don't think we should apply styles to generic tag names.

Reply:
See above.